driver/cam_hal.c

```diff
@@ -365,7 +365,7 @@ esp_err_t cam_config(const camera_config_t *config, framesize_t frame_size, uint
     cam_obj->height = resolution[frame_size].height;
 
     if(cam_obj->jpeg_mode){
-        cam_obj->recv_size = cam_obj->width * cam_obj->height / 5;
+        cam_obj->recv_size = cam_obj->width * cam_obj->height / 5 + CONFIG_JPEG_FORMAT_EXTRA_BUFFER_SIZE;
```

Comment:
The person setting this parameter should not like the addition operation

Reply:
If there is no error, we don't need to care about the addition operation at all. If an error occurs, we can change this value to avoid the error.